src/components/oculus-touch-controls.js

```diff
@@ -150,6 +150,7 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
       controller: 0,
       rotationOffset: data.rotationOffset !== -999 ? data.rotationOffset : isRightHand ? -90 : 90
     });
+    el.emit('controller-connected', {type: 'oculus-touch-controller'});
```

Comment:
won't work as there is no context provided to distinguish which component injected the tracked-controls.  if that is provided to `tracked-controls` when injected, then I agree e.g. by expanding schema with something like `source` that is name of injecting component `oculus-touch-controls`, `vive-controls`, `daydream-controller` (which maybe should have been -controls), `gearvr-controls` and then providing `source` in the `controllerconnected` data